che_devfile/structure.go

```diff
@@ -0,0 +1,107 @@
+package che_devfile
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v2"
+)
+
+// Tools define Che plugin
+type Tools struct {
+	Name        string `yaml:"name"`
```

Comment:
If you use the YAML library that I presented earlier, you only have to define struct tags for json and they will be re-used for YAML as well.